src/main/java/io/javalin/core/compression/DynamicCompressionStrategy.kt

```diff
@@ -0,0 +1,58 @@
+package io.javalin.core.compression
+
+import org.meteogroup.jbrotli.libloader.BrotliLibraryLoader
+import io.javalin.Javalin.log
+import io.micrometer.core.lang.Nullable
+import javax.validation.constraints.Null
+
+/**
+ * This class is a settings container for Javalin's dynamic content compression.
+ *
+ * It is used by DynamicCompressionHandler to determine the encoding and parameters that should be used
+ * when encoding a dynamic response from the server.
+ *
+ * @see DynamicCompressionHandler
+ *
+ * @param brotli instance of Brotli handler, default = null
+ * @param gzip   instance of Gzip handler, default = null
+ */
+class DynamicCompressionStrategy(brotli: Brotli? = null, gzip: Gzip? = null) {
```

Comment:
This class looks great now!

Reply:
Glad you agree! I think so too. Lots less junk in there